src/datumaro/plugins/openvino_plugin/samples/googlenet-v4-tf_interp.py

```diff
@@ -29,11 +31,20 @@ def preprocess(self, inp: DatasetItem) -> Tuple[LauncherInputType, PrepInfo]:
         return img, None
 
     def postprocess(self, pred: ModelPred, info: PrepInfo) -> List[Annotation]:
+        logit = pred.get(self.LOGIT_KEY)
+        if logit is None:
+            raise DatumaroError(f'"{self.LOGIT_KEY}" key should exist in the model prediction.')
```

Comment:
```suggestion
        logit = pred.get(self.LOGIT_KEY, [])
        if not logit:
            raise DatumaroError(f'"{self.LOGIT_KEY}" key should exist in the model prediction.')
```
It seems better to use default values for safer.

Reply:
thank you. let me update this later.